### ISSUE TITLE
Printing to PDF in Chrome, macOS seems to skip fields

### ISSUE THREAD

**Issue by ryanhofdotgov** [18F/e-QIP-prototype#817]
**What did you do?**

Enter foreign relationships --> foreign contact. Go to review page for that section. Expand everything that can be expanded (e.g., table rows). Print to PDF.

**What did you expect to see?**

PDF with all the fields populated.

**What did you see instead?**

Missing first name field. 

**Additional Information**

See attached test case PDF which was created by going to review page of eApp section 6.  Page 4 shows first name as being blank, but the data was entered as John. (See full name at bottom of page 3).

Using latest Chrome, macOS 10.12.6.

I'm wondering if it is some kind of CSS discrepancy on this particular field (i.e., something special about it, vs others).

[6.pdf](https://github.com/18F/e-QIP-prototype/files/2393372/6.pdf)

**Comment by ryanhofdotgov** [comment]
Same thing happening in Foreign Business Venture sub-section. See second test case PDF attached below. Page 77. On Page 76, you can see that the first name is Luc, but the field is empty on page 77. Both known issues of this problem is happening in eApp section 6, on 2 separate test cases (#2 and #5).

[section6.pdf](https://github.com/18F/e-QIP-prototype/files/2393407/section6.pdf)

**Comment by line47** [comment]
@ryanhofdotgov I was not able to reproduce the issue - I entered in first names here and they shows up with printing to PDF on that section. Names are `James A Brown` and `Jimmy A Simons`

PDF attached
[testingForeignContacts-names.pdf](https://github.com/18F/e-QIP-prototype/files/2406237/testingForeignContacts-names.pdf)

**Comment by ryanhofdotgov** [comment]
@line47 Can you checkout the branch on this yet-to-be-merged PR (https://github.com/18F/e-QIP-prototype/pull/842) and then load up test3.json per the documentation listed here:
https://github.com/18F/e-QIP-prototype/blob/45b12b43b947e88d1ddc041421ea2d2be9281bb2/docs/test-scenarios.md
Then see if you can re-produce? 

Shoot me a DM if you get stuck or want to pair on loading the test data in.

**Comment by ryanhofdotgov** [comment]
Since Scott can't reproduce, I'm going to take this issue and see if I can still replicate or perhaps find  a different way of producing the PDFs.

**Comment by ryanhofdotgov** [comment]
Un-assigning myself, in case someone else has time to try and reproduce. Goal is to be able to go to all 10 review sections of eApp and create PDFs of that page, without any missing fields, in order to get a "good" archival PDF out of eApp.

**Comment by ryanhofdotgov** [comment]
Some thoughts I had: Chrome under Windows 10, some kind of print to PDF function, Firefox under macOS, etc.

And to clarify: For every test case I provide to NBIB to review, I have to manually produce this "archival" PDF by creating one big PDF that is the concatenation of the print-to-PDF of each section's review page. This is a work-around process for #701.

**Comment by ryanhofdotgov** [comment]
I prepared 4 more archival PDFs, scanning through each page for the missing fields NBIB originally saw. I did not see the missing fields any longer. 

I'm not sure it made a difference, but this round, I directly saved each section review page to PDF via the Chrome its print dialog instead of using the system print dialog. 

Next I concatenated each set of 10 PDFs by opening all of them simultaneously from the Finder with Preview and printing to a single PDF. Each PDF was named 1.pdf, 2.pdf, 3.pdf, so Preview ordered them correctly.
